yarn.lock

```diff
@@ -0,0 +1,612 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
- That yarn.lock file -> the package currently works using a package-lock.json file, which hints we should use npm to handle package versioning. If this package decides to move to yarn, that means we should delete the package-lock.json file and make it clear that we should use yarn.
Either way, this discussion is out of place in this merge request, so you should stick to using npm, and test your edits with npm's lock file current versions.

Reply:
I will remove yarn.lock soon.